app/src/docs/v1.api-spec.yaml

```diff
@@ -17,30 +17,348 @@ security:
   - BearerAuth: []
     OpenID: []
 paths:
-  /hello:
+  '/':
     get:
-      summary: Returns hello world result
-      operationId: getHello
+      summary: List all user accessible objects
+      description: >-
+        This endpoint returns a list of all user accessible objects.
+      operationId: GetAccessibleObjects
+      tags:
+        - Object
+      responses:
+        '200':
+          description: Returns the object information
+          content:
+            application/json:
+              schema:
```

Comment:
this is still WIP.. but i think it will retunr an array so maybe change to:
```                
type: array
items:
  $ref: '#/components/schemas/ObjectDetails'
```

Reply:
I think the only ones that will likely come back as an array are the following endpoints:
`POST /object`
`GET /object/{objId}/versions`
The rest of them are likely going to be top level objects instead of arrays - still tentative though.

Reply:
Changed Description and type for these endpoints